internal/core/cluster/sbd_test.go

```diff
@@ -55,98 +56,19 @@ sbd failed; please check the logs.`
 	return []byte(output)
 }
 
-func (suite *SbdTestSuite) TestSbdDump() {
```

Comment:
Removing this tests as this part of the code is already tested in other public function tests